python/GafferUI/ColorChooser.py

```diff
@@ -44,24 +44,55 @@
 
 from Qt import QtGui
 
+__tmi2rgbMatrix = imath.M33f(
+	-1.0 / 2.0, 0.0, 1.0 / 2.0,
+	1.0 / 3.0, -2.0 / 3.0, 1.0 / 3.0,
+	1.0, 1.0, 1.0
+)
+__rgb2tmiMatrix = __tmi2rgbMatrix.inverse()
+
+def _tmi2rgb( c ) :
```

Comment:
`tmiToRGB` and `rgbToTMI` perhaps?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5962/commits/025b13f3e7761a8e6e3147b22ffb1f8e728548bf